frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/utils/behavior-to-german-name.pipe.ts

```diff
@@ -15,6 +15,7 @@ const behaviorTypeToGermanNameDictionary: {
     requestBehavior: 'Fahrzeuge anfordern',
     transferBehavior: 'Fahrzeuge versenden',
     transferToHospitalBehavior: 'Patienten abtransportieren',
+    managePatientTransportToHospitalBehavior: 'Patientenabtransport verwalten',
```

Comment:
We could name this `Transportorganisation` since this name is already established in the domain.